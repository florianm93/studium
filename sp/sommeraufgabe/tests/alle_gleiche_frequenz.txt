**
**    Alle Sender k√∂nnen die gleiche Frequenz benutzen
**
3.5 3.5 1.5
7 7 1.5
0 0 1.5
0 7 1.5
7 0 1.5