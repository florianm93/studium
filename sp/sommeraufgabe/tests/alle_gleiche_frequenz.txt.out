**
** Daten aus Datei: tests/alle_gleiche_frequenz.txt
**

Senderpositionen (X,Y) und Senderradien:
S01:   3.500,   3.500,   1.500
S02:   7.000,   7.000,   1.500
S03:   0.000,   0.000,   1.500
S04:   0.000,   7.000,   1.500
S05:   7.000,   0.000,   1.500

Frequenzzuordnung:
Anzahl benoetiger Frequenzen: 1
 Frequenz | Sender
=======================================================================================
    0     |  S01  S02  S03  S04  S05  
